style.css

```
@@ -0,0 +1,49 @@
+* {
+    font-family: sans-serif;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background: #0a0a23;
+}
+
+.container {
+    position: relative;
+    max-width: 600px;
+    height: 500px;
+    padding: 0px;
+    color: black;
+    text-align: center;
+}
+
+.language-section {
+    background: #ffffff;
+}
+
+.button {
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: 0 15px;
+    font-weight: bold;
+    background: #feac32;
+    color: #0a0a23;
+    font-size: 22px;
+
+}
+
+.title {
+    color: white;
```

You are using `white` here and `#ffffff` below. Be consistent with your color declarations in your CSS.
```suggestion
    color: #FFF;
```